docs/pages/tutorial/introduction.mdx

```diff
@@ -73,7 +73,7 @@ const styles = StyleSheet.create({
 >
 > <br />
 > Go ahead and press the above button. You will see the above code running in a Snack. Try to switch
-> between iOS, Android, or the web tabs. You can also open it on your device in the Expo Go app from
+> between Android, iOS or the web tabs. You can also open it on your device in the Expo Go app from
```

Comment:
Snack also uses iOS, Android, Web order so let's stick to it here in content of this info supporting the actual UI of the platform.

Reply:
Let's change the order in Snack then too, so it matches our guidelines. 😉